endpoint/templates/name.model.spec(mongooseModels).js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var app = require('../../app');
+var <%= classedName %> = require('./<%= name %>.model');
+var <%= name %>;
```

Comment:
Also, how could I check what Travis is doing? I mean 

```javascript

  28 passing (305ms)
  2 failing
  1) Foo Model should begin with no Foos:
     TypeError: undefined is not a function
      at Context.<anonymous> (server/api/Foo/Foo.model.spec.js:30:16)
  2) Foo Model "after each" hook:
     TypeError: undefined is not a function
      at Context.<anonymous> (server/api/Foo/Foo.model.spec.js:26:16)
```

This doesn't tell me much, not sure where to search the error